### ISSUE TITLE
Status?

### ISSUE THREAD

**Issue by jbenet** [fd/webrtc#1]
Hey @fd wondering what the status here is?

**Comment by fd** [comment]
Mostly experimental right now. It works but building everything is a pain.

**Comment by fd** [comment]
Also openwebrtc might be a better backend.

**Comment by jbenet** [comment]
wow! I don't even know how that escaped me.

**Comment by fd** [comment]
Openwebrtc looks very promising indeed. I'm probably going to use it for telehash instead of libjingle. But currently it lacks datachannels so that will have to wait.

**Comment by jbenet** [comment]
@fd: DataChannel support merged 8 days ago into openwebrtc. https://github.com/EricssonResearch/openwebrtc/issues/3

**Comment by fd** [comment]
I haven't had the chance to investigate the code yet.

**Comment by jbenet** [comment]
@fd https://groups.google.com/d/msg/golang-nuts/EHZtnPxdopU/Tb0T1VZZuVAJ

**Comment by dmitshur** [comment]
> It works but building everything is a pain.

Indeed. Building the [`webrtc`](https://code.google.com/p/webrtc/) library is not as easy as `go build`. I'm trying to figure it out now.

**Comment by hackwaly** [comment]
How to install this package?
I got:

```
root@localhost:~# go get github.com/fd/webrtc
# pkg-config --cflags libwebrtc
Package libwebrtc was not found in the pkg-config search path.
Perhaps you should add the directory containing `libwebrtc.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libwebrtc' found
exit status 1
```

**Comment by fd** [comment]
You have to `cd` onto the package directory and run `make`.

**Comment by hackwaly** [comment]
There are 2800+ source files need to compile, compare to node-webrtc is 310+.
Are there some unnecessary dependencies could be omit?

**Comment by amitu** [comment]
Libjingle seems to have moved to chromium code base now. node-webrtc uses that one.

This one works, but can not send data on DataChannel over 1kb per packet or so, where as with node-webrtc I can send data upto 65k or so. Also it does not have any sort of hint when to send data in case .Send fails. .Send returns a bool saying if data was sent or not, but putting a sleep and retrying is really the worst way to code. There is no event which says data was already sent, not send more etc. 

Am trying to get it to build with chromium version of libjingle (assuming this behaviour is due to libjingle@ deficiencies). Let me know if you have tried this, so I dont waste time.

**Comment by mjena** [comment]
make command fails with following error:
OSError: [Errno 2] No such file or directory: '/Users/maheshjena/Work/WorkSpace/src/github.com/fd/webrtc/build/darwin_amd64/src/libwebrtc/_gclient_trunk_LR84v8'
make: *** [build-libwebrtc] Error 1

Anything to add for this?

**Comment by fd** [comment]
@mjena You will probably want to use https://github.com/keroserene/go-webrtc
